sdk/monitor/azquery/utils_test.go

```diff
@@ -164,12 +133,25 @@ func startMetricsTest(t *testing.T) *azquery.MetricsClient {
 	return client
 }
 
-func lookupEnvVar(s string) string {
-	ret, ok := os.LookupEnv(s)
-	if !ok {
-		panic(fmt.Sprintf("Could not find env var: '%s'", s))
+func getEnvVar(lookupValue string, fakeValue string) string {
+	// get value
+	envVar := fakeValue
```

Comment:
Nitpicking, I'd change a couple names here to make clear that we're looking up the value of an environment variable and returning either that value or a fake one
```suggestion
func getEnvVar(envVar string, fakeValue string) string {
	// get value
	value := fakeValue
```

Reply:
Fixed in [d0c65c0](https://github.com/Azure/azure-sdk-for-go/pull/21965/commits/d0c65c005478be252e03a390bdc8d4ded5ffe04a)